docs/deployment/apify_platform.mdx

```diff
@@ -0,0 +1,254 @@
+---
+id: apify-platform
+title: Apify Platform
```

Comment:
AFAIK it should be "Apify platform" (platform is lower case, only Actor is uppercased). Please fix all occurrences.